### ISSUE TITLE
result question

### ISSUE THREAD

**Issue by cavehuangly** [JCruan519/VM-UNet#99]
why?
val epoch: 30, loss: 0.9849, miou: 0.0, f1_or_dsc: 0.0, accuracy: 0.9901655296578792,                 specificity: 1.0, sensitivity: 0.0, confusion_matrix: [[41076312        0]
 [  407976        0]]

<img width="1037" alt="Image" src="https://github.com/user-attachments/assets/7d0e7c99-2bea-4c02-8ca7-cfaa6b11eb4f" />

**Comment by f-kuzey-edes-huyal** [comment]
Have you calculated the mean and standard deviation of your training images? It may help. You need to insert your dataset’s mean and standard deviation in the utils file within the normalization class. The criterion might also be an issue—try setting 0.7 for Dice loss and 0.3 for binary cross-entropy. That might help as well.

**Comment by Defrauder** [comment]
Your train epochs are too less.
